### ISSUE TITLE
[feature request] verifier - Allow disambiguating multiple occurrences of an anchor match [phab:T138]

### ISSUE THREAD

**Issue by kythe** [google/kythe#1237]
_Created by **pludemann** at 2016-06-10 00:29:27_: 

For templates and parametric declarations, it's often the case the parameter appears multiple times in a line.  e.g.

```
def max(arg1: T, arg2: T) -> T
```

The verifier doesn't allow `"T"` because it's ambiguous, which requires a clumsy (and error-prone) notation with absolute offsets. I suggest something like `#n` to match the "n"th item (leaving open the question of whether 0-origin or 1-origin is best):
```
#- MaxArg1Type=#1"T" ref vname("max.<template_arg>.T", "corpus", "", "", "python")
#- MaxArg2Type=#2"T" ref(vname("max.<template_arg>.T", "corpus", "", "", "python")
#- MaxReturnType=#3"T" ref(vname("max.<template_arg>.T", "corpus", "", "", "python")
def max(arg1: T, arg2: T) -> T

**Comment by kythe** [comment]
_**pludemann** wrote at 2016-06-21 20:07:50_:

It seems that my request has been answered by
#1244

**Comment by kythe** [comment]
_**zarko** wrote at 2016-06-21 20:48:00_:

Resolved by #1244.
